docs/xlp.md

```diff
@@ -31,13 +31,19 @@ Requests the Eclipse OpenJ9&trade; VM to allocate the Java&trade; object heap an
 
 If you use the [`-Xgc:preferredHeapBase`](xgc.md#preferredheapbase) option with `-Xlp`, the preferredHeapBase address must be a multiple of the large page size. If you specify an inaccurate heap base address, the heap is allocated with the default page size.
 
-To find out the large page sizes available and the current setting, use the `-verbose:sizes` option. Note that the current settings are the requested sizes and not the sizes obtained. For object heap size information, check the `-verbose:gc` output.
+To find out the large page sizes available and the current setting, use the `-verbose:sizes` option. These current settings are the requested sizes and not the sizes obtained. For object heap size information, check the `-verbose:gc` output.
+
+Configuring the large page memory allocation after you enable the large page support is different in different platforms. For more information, see [Configuring large page memory allocation](configuring.md#configuring-large-page-memory-allocation).
+
+If the configured large page size is greater than the size of the total code cache for JIT, then the page size that is used for code cache allocation is recalculated. The next available lower page size on the platform is identified and used for the code cache allocation.
+
+For example, if the large page size is configured as 1 GB and the total JIT code cache size is not modified (by using the [`-Xcodecachetotal`](xcodecachetotal.md) option), then the code cache size will be the default JIT code cache size, 256 MB (for a 64-bit VM). Thus, the next available page size lower than 256 MB is used for code cache allocation. If the next available lower size page size is 2 MB, then 128 pages (of 2 MB each) are allocated for the code cache.
```

Comment:
This needs to be rephrased. Currently it reads:
if  (1) large page size is 1GB   AND
    (2) default code cache size is not modified
then
    (3) code cache size is 256 MB
In reality (2) ==> (3)

How about something along these lines:
For example, on a system where 1 GB, 2 MB and 4 KB pages are available, the VM will refrain from using 1 GB pages for the code cache, because even a single 1GB page would result in exceeding the total size of the code caches (256 MB for 64-bit systems, assuming it wasn't changed with the [`-Xcodecachetotal`](xcodecachetotal.md) option). Instead, the VM will pick the next lower page size (2 MB in this example) that satisfies the total code cache size requirement.